python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled.py

```diff
@@ -0,0 +1,415 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+  #####################################
+  ##           Gherkin               ##
+  #####################################  
+  Rule Name:
+  BUSINESS_SUPPORT_OR_ABOVE_ENABLED
+  
+  Description:
+  Check whether the AWS Account has Business or above(Enterprise) support plan  is enabled
+  
+  Trigger:
+  Frequency every 3hours
+  
+  Reports on:
+  AWS::::Account
+  
+  Rule Parameters:
+  None
+  
+  Scenarios:
+  Scenario: 1
+    Given: AWS Account has ability to use Support API's DescribeCases call with results returned without ClientError Exception
+     Then: Return COMPLIANT
+  Scenario: 2
+    Given: AWS Support API call errors out with SubscriptionRequired exception while using the DescribeCases API call. This error means Business Plan is not currently used for the account
+     Then: Return NON_COMPLIANT
+  '''
+import json
+import sys
+import datetime
+import boto3
+import botocore
+from botocore.exceptions import ClientError
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+    """Form the evaluation(s) to be return to Config Rules

```

Comment:
Remove those explanations. It is only an help for the early developers.

Reply:
removed explanation under evaluate_compliance